server/controller/authController.js

```diff
@@ -1,5 +1,9 @@
 const { validationResult } = require("express-validator");
 const bcrypt = require("bcryptjs");
+const nodemailer = require("nodemailer");
+const jwtSecret = "HaHa";
```

Comment:
There is no need to intialise again jwtSecret here.
Use methods define in config/secret.js.